templates/event_agent_delete.conf.j2

```diff
@@ -0,0 +1,14 @@
+# {{ ansible_managed }}
+[event-agent]
+tube = {{ openio_event_agent_tube_delete }}
+namespace = {{ openio_event_agent_namespace }}
+user = openio
+queue_url = {{ openio_event_agent_queue_url }}
+workers = 1
+concurrency = 1
+handlers_conf = /etc/oio/sds/{{ openio_event_agent_namespace }}/{{ openio_event_agent_servicename }}/oio-event-handlers-delete.conf
+log_facility = LOG_LOCAL0
+log_level = info
+log_name = {{ openio_event_agent_servicename }}
+log_address = /dev/log
+syslog_prefix = OIO,{{ openio_event_agent_namespace }},oio-event-agent-delete,{{ openio_event_agent_serviceid }}
```

Comment:
why change the syslogtag ?

Reply:
Please dont. The syslogtag is based on the service name for all services.